src/ui/listview.rs

```diff
@@ -72,7 +72,9 @@ impl<I: ListItem> ListView<I> {
             library,
             pagination: Pagination::default(),
             title: "".to_string(),
-        }
+        };
+        result.try_paginate();
```

Comment:
Hmm, could you explain why this change is necessary?

Reply:
When the page loads on an ultrawide screen for example, it may not show enough items initially to fill the screen. This makes sure that doesn't happen. Maybe there's a better way to solve this...